packages/web/app/src/components/v2/changelog/changelog.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { ReactElement, useEffect, useState } from 'react';
+import { Button } from '@/components/v2';
```

Comment:
We don't use these components anymore, please don't use any component from `@/components/v2`, use `@/components/ui` instead. Also, please don't place any new components within the  `v2` folder. Our final goal is to remove that folder.